src/components/TouchableWithoutFocus.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Element that should be clickable  */
+    children: PropTypes.oneOfType([
+        PropTypes.node,
+        PropTypes.func,
+    ]).isRequired,
+
+    /** Callback for onPress event */
+    onPress: PropTypes.func.isRequired,
+
+    /** Styles that should be passed to touchable container */
+    styles: PropTypes.arrayOf(PropTypes.object),
+};
+
+const defaultProps = {
+    styles: [],
+};
+
+/**
+ * This component prevents the tapped element from capturing focus
+ */
+class TouchableWithoutFocus extends React.Component {
```

Comment:
NAB: this component could be a functional component since there is no state being used. The `pressAndBlur` method could be defined inside the render function. I don't feel too strongly about making this change, but I think it's worth considering.

Reply:
Made it a class component because of the ref usage.